concrete-core/src/specification/engines/lwe_ciphertext_inplace_storing.rs

```diff
@@ -0,0 +1,48 @@
+use super::engine_error;
+use crate::specification::engines::AbstractEngine;
+use crate::specification::entities::{LweCiphertextEntity, LweCiphertextVectorEntity};
+use concrete_commons::parameters::LweCiphertextIndex;
+
```

Comment:
Hmm this one I am not sure. I think I tried to follow the convention `Object(Discarding|Fusing||)VerbAsNounEngine` such that sentences of the form "This engine can `Verb` a `Object`" would represent what the trait does. I belive it is the case here, because the engine stores a ciphertext inside a ciphertext vector.